src/content/docs/contributing/client-api/rbac.v2.mdx

```diff
@@ -0,0 +1,197 @@
+        ```
+      params:
+        - name: resourceId
+          type:
+            name: string
+          description: The workspace identifier
+      returns:
+        name: ResourceReference
+        link: https://buf.build/project-kessel/inventory-api/docs/main:kessel.inventory.v1beta2#kessel.inventory.v1beta2.ResourceReference
+        description: A ResourceReference for the workspace
+
+    - name: principalSubject
+      description: |
+        Creates a SubjectReference for a user principal based on user ID and domain.
+        This is a convenience function that wraps principalResource to create a subject reference.
+
+        Example:
+
+        ```python
+        userSubject = principalSubject("john.doe", "example.com")
+        ```
+      params:
+        - name: id
+          type:
+            name: string
+          description: The user identifier
+        - name: domain
+          type:
+            name: string
+          description: The domain or organization the user belongs to
+      returns:
+        name: SubjectReference
+        link: https://buf.build/project-kessel/inventory-api/docs/main:kessel.inventory.v1beta2#kessel.inventory.v1beta2.SubjectReference
+        description: A SubjectReference for the user's principal
+
+    - name: subject
+      description: |
+        Creates a SubjectReference from a ResourceReference and an optional relation.
+        This function allows you to easily create a subject reference.
+
+        Example:
+
+        ```python
+        # Create a subject reference from a principal resource with a relation
+        memberSubject = subject(principalResource("123", "redhat"), "member")
+        
+        # Create a subject reference without a relation (direct subject)
+        directSubject = subject(principalResource("456", "redhat"))
+        
+        # Create a subject reference using a manually constructed ResourceReference
+        customResource = ResourceReference(
+            resourceType="group",
+            resourceId="our-team",
+            reporter=None
+        )
+        groupSubject = subject(customResource, "owner")
+        ```
+      params:
+        - name: resourceRef
+          type:
+            name: ResourceReference
+            link: https://buf.build/project-kessel/inventory-api/docs/main:kessel.inventory.v1beta2#kessel.inventory.v1beta2.ResourceReference
+          description: The resource reference that identifies the subject
+        - name: relation
+          type:
+            name: string
+          description: Optional relation that points to a set of subjects (e.g., "members", "owners")
+          optional: true
+      returns:
+        name: SubjectReference
+        link: https://buf.build/project-kessel/inventory-api/docs/main:kessel.inventory.v1beta2#kessel.inventory.v1beta2.SubjectReference
+        description: A reference to a Subject or, if a relation is provided, a Subject Set.
+
+    - name: listWorkspaces
```

Comment:
This touches client specific API - I think we should have the `async`  version, similar to the [buildAsync](https://github.com/project-kessel/docs/blob/8fbfd36882f004343e6bc62c36cd0a9e1dedbe34/src/content/docs/contributing/client-api/service-version.mdx?plain=1#L108-L120).